### PR TITLE
jsonnet-bundler: init at 0.4.0

### PR DESCRIPTION
**Pull request by Preisschild**
###### Things done

- [ ] Tested using sandboxing ([nix.useSandbox](https://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](https://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [x] Tested compilation of all pkgs that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [x] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [x] Ensured that relevant documentation is up to date
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

### CODE REVIEW

File: maintainers/maintainer-list.nix [hunk 1]
```diff
@@ -9834,4 +9834,10 @@
     github = "hloeffler";
     githubId = 6627191;
   };
+  preisschild = {
```
Comment (SuperSandro2000) [inline maintainers/maintainer-list.nix:9837]: Can you put this into a separate commit per Contribution Guide?

File: pkgs/development/tools/jsonnet-bundler/default.nix [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+  src = fetchFromGitHub {
+    owner = "jsonnet-bundler";
+    repo = "jsonnet-bundler";
+    rev = "v${version}";
+    sha256 = "0pk6nf8r0wy7lnsnzyjd3vgq4b2kb3zl0xxn01ahpaqgmwpzajlk";
+  };
+
+  vendorSha256 = "1h1d32zi1pfb7spprsacxg8ygrigswqqngrc8gsdzn1xbldphl5g";
+
+  doCheck = false;
```
Comment (SuperSandro2000) [inline pkgs/development/tools/jsonnet-bundler/default.nix:16]: Can you add a comment why checks are disabled?

**Review by SuperSandro2000 — CHANGES_REQUESTED** [review]

File: maintainers/maintainer-list.nix (hunk 1 above)
Comment (Preisschild) [reply maintainers/maintainer-list.nix]: Done. Thanks for the information, must have read over it.

File: pkgs/development/tools/jsonnet-bundler/default.nix (hunk 2 above)
Comment (Preisschild) [reply pkgs/development/tools/jsonnet-bundler/default.nix]: Ah thanks for the reminder, tests are working and are enabled now.

**Comment by SuperSandro2000** [comment]
Result of `nixpkgs-review pr 101865` [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>1 package failed to build:</summary>
  <ul>
    <li>jsonnet-bundler</li>
  </ul>
</details>

**Comment by Preisschild** [comment]
> Run on darwin:
> 
> Result of `nixpkgs-review pr 101865` [1](https://github.com/Mic92/nixpkgs-review)
> 1 package failed to build:

Thanks for the information, I'll check tomorrow if i can reproduce on darwin.

File: pkgs/development/tools/jsonnet-bundler/default.nix [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+  vendorSha256 = "1h1d32zi1pfb7spprsacxg8ygrigswqqngrc8gsdzn1xbldphl5g";
+
+  doCheck = true;
+
+  subPackages = [ "cmd/jb" ];
+
+  buildFlagsArray = [ "-ldflags=-s -w -X main.Version=${version}" ];
+
+  meta = with lib; {
+    description = "A jsonnet package manager.";
```
Comment (sikmir) [inline pkgs/development/tools/jsonnet-bundler/default.nix:23]:
```suggestion
    description = "A jsonnet package manager";
```
`meta.description` should not have a period at the end, see [contributing guide](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

**Comment by SuperSandro2000** [comment]
Result of `nixpkgs-review pr 101865` [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>1 package built:</summary>
  <ul>
    <li>jsonnet-bundler</li>
  </ul>
</details>

**Comment by Preisschild** [comment]
Nice, just tested it on darwin too, all tests are OK and the program works.

File: pkgs/development/tools/jsonnet-bundler/default.nix [hunk 4]
```diff
@@ -0,0 +1,28 @@
...
+  src = fetchFromGitHub {
+    owner = "jsonnet-bundler";
+    repo = "jsonnet-bundler";
+    rev = "v${version}";
+    sha256 = "0pk6nf8r0wy7lnsnzyjd3vgq4b2kb3zl0xxn01ahpaqgmwpzajlk";
+  };
+
+  vendorSha256 = null;
+
+  doCheck = true;
```
Comment (sikmir) [inline pkgs/development/tools/jsonnet-bundler/default.nix:16]: `doCheck` is true by default:
https://github.com/NixOS/nixpkgs/blob/3d04e9a779a3ca28230e0e8f3c4310264960e979/pkgs/development/go-modules/generic/default.nix#L211

Comment (Preisschild) [reply pkgs/development/tools/jsonnet-bundler/default.nix]: Thanks, I removed it.

**Comment by Preisschild** [comment]
sorry for adding some people as reviewer, i had a bad rebase

**Review by sikmir — APPROVED** [review]

**Comment by SuperSandro2000** [comment]
Result of `nixpkgs-review pr 101865` run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>1 package built:</summary>
  <ul>
    <li>jsonnet-bundler</li>
  </ul>
</details>

**Comment by SuperSandro2000** [comment]
Result of `nixpkgs-review pr 101865` run on x86_64-darwin [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>1 package built:</summary>
  <ul>
    <li>jsonnet-bundler</li>
  </ul>
</details>
